doc/release-notes.md

```diff
@@ -61,6 +61,15 @@ platform.
 Notable changes
 ===============
 
+New RPCs
+--------
+
+- `getbalances` returns an object with all balances (`mine`,
+  `untrusted_pending` and `immature`). Please refer to the RPC help of
+  `getbalances` for details. The new RPC is intended to replace
```

Comment:
Good call on referring to the RPC help for details. This is what the release notes should do instead of substitute for documentation :+1: